src/server/helpers/utils.ts

```diff
@@ -361,3 +361,7 @@ export async function parseInitialState(req, type):Promise<Record<string, any>>
 export function parseQuery(url: string) {
 	return new URLSearchParams(url.replace(/^.+?\?/, ''));
 }
+
+export function getIntFromQueryParams(query: URLSearchParams, type: string, fallback = 0) {
+	return parseInt(query.get(type), 10) || fallback;
+}
```

Comment:
Nitpick: The parameter is usually called `name` to follow the [specification](https://url.spec.whatwg.org/#interface-urlsearchparams) :nerd_face:
```suggestion
export function getIntFromQueryParams(query: URLSearchParams, name: string, fallback = 0) {
	return parseInt(query.get(name), 10) || fallback;
}
```